themes/compose/assets/sass/_variables.sass

```diff
@@ -9,7 +9,7 @@ html
   --overlay: var(--light)
   --text: var(--dark)
   --theme: #20c997
-  --font: 'Metropolis',-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji
+  --font: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
```

Comment:
Btw I think you still need to remove the @font-face declarations in some of the other saas files. This will prevent the browser trying to pull the resources.

Reply:
And also remove the source files for the fonts